### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by emmenko**
Closes #606 
Closes #607 
Closes #608 
Closes #609 
Closes #610 
Closes #611

### CODE REVIEW

**Comment by emmenko** [comment]
Right, I see that now in the logs. Thanks for the reminder!

File: scripts/get-babel-preset.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ module.exports = function getBabelPreset() {
           targets: {
             browsers: ['last 1 versions'],
           },
+          corejs: 2,
```
Comment (tdeekens) [inline scripts/get-babel-preset.js]: We also need to make the dependency explicit and have it in our pkg.json.

Comment (emmenko) [reply scripts/get-babel-preset.js]: Hmm are you sure? It seems to work like this..anyway, I can add it

Comment (tdeekens) [reply scripts/get-babel-preset.js]: Yes, pretty much.

1. https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#babelpolyfill

> Right now @babel/polyfill is mostly just an alias of core-js v2

I would maybe remove @babel/polyfill and make core-js@2 explicit?

Comment (emmenko) [reply scripts/get-babel-preset.js]: We don't use `@babel/polyfill`. I added the `core-js` as a dependency. Is that ok now?

File: .travis.yml [hunk 2]
```diff
@@ -11,6 +11,11 @@ cache:
   directories:
     - node_modules
 
+before_install:
+  # Install yarn
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.15.2
+  - export PATH=$HOME/.yarn/bin:$PATH
```
Comment (tdeekens) [inline .travis.yml]: Will add this to app-kit.

Comment (emmenko) [reply .travis.yml]: We have it already there, you can bump the yarn version though 😉

Comment (tdeekens) [reply .travis.yml]: Did that.

**Pull request merged**